README.md

```diff
@@ -118,7 +118,14 @@ When creating a new actor, `create-new-actor.sh` will generate a new test case
 template to ensure the new actor can run against different MongoDB topologies,
 please update the template as needed so it uses the newly created actor.
 
-### Debugging
+## Patch-Testing and Evergreen
+
+When restarting any of Genny's Evergreen tasks, make sure you restart
```

Comment:
s/tasks/self-tests? Avoid ambiguity with sys-perf genny tasks (which admittedly don't contain the word "genny", but still)